src/app/pages/TokenDashboardPage/hook.ts

```diff
@@ -46,21 +61,19 @@ export const useTokenTransfers = (scope: SearchScope, address: string) => {
     // Loading transactions on the consensus layer is not supported yet.
     // We should use useGetConsensusTransactions()
   }
-
-  const oasisAddress = useTransformToOasisAddress(address)
   const query = useGetRuntimeEvents(
     network,
     layer, // This is OK since consensus has been handled separately
     {
       ...pagination.paramsForQuery,
-      rel: oasisAddress!,
-      type: 'evm.log',
+      type: RuntimeEventType.evmlog,
       // The following is the hex-encoded signature for Transfer(address,address,uint256)
       evm_log_signature: 'ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
+      ...params,
     },
     {
       query: {
-        enabled: !!oasisAddress,
+        enabled: !!params.rel || (!!params.contract_address && !!params.nft_id),
```

Comment:
oh :/

Reply:
yes, we are enabling queries when oasis address is returned by `useTransformToOasisAddress`. How do you want to change it?